### PR TITLE
(fix) default browse path for glossary term

### PR DESCRIPTION
**Pull request by saxo-lalrishav**
## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [x] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

**Comment by saxo-lalrishav** [comment]
#2787

File: metadata-models/src/main/pegasus/com/linkedin/metadata/aspect/GlossaryTermAspect.pdl [hunk 1]
```diff
@@ -4,8 +4,10 @@ import com.linkedin.glossary.GlossaryTermInfo
...
 import com.linkedin.common.Status
+import com.linkedin.common.BrowsePaths
+
 
 /**
  * A union of all supported metadata aspects for a CorpUser
  */
-typeref GlossaryTermAspect = union[GlossaryTermKey, GlossaryTermInfo, Ownership, Status]
\ No newline at end of file
+typeref GlossaryTermAspect = union[GlossaryTermKey, GlossaryTermInfo, Ownership, Status,BrowsePaths]
```
Comment (gabe-lyons) [inline metadata-models/src/main/pegasus/com/linkedin/metadata/aspect/GlossaryTermAspect.pdl:13]: nit: add a space after `Status,` for consistency

**Review by gabe-lyons — APPROVED** [review]

**Comment by gabe-lyons** [comment]
@saxo-lalrishav looks like you need to re-run gradle build and commit the newly generated avro schemas

**Review by shirshanka — APPROVED** [review]
LGTM!

**Pull request merged**